zerver/data_import/rocketchat.py

```diff
@@ -670,7 +670,7 @@ def message_to_dict(message: Dict[str, Any]) -> Dict[str, Any]:
                 message_dict["recipient_id"] = huddle_id_to_recipient_id[huddle_id]
             else:
                 rc_member_ids = direct_id_to_direct_map[rc_channel_id]["uids"]
-                if rc_sender_id == rc_member_ids[0]:
+                if rc_sender_id == rc_member_ids[0] and len(rc_member_ids) >= 2:
```

Comment:
Hmm, could this just be an RC private message to yourself? I think if so, we'd prefer to not drop those, and instead have an `if len(rc_member_ids) == 1` block just above this one.

(or is it 0? I guess we'd need to skip those).

Reply:
This has been adressesed in the meantime by e653bb2733db729480c09bb36dfacf5a5fb8ae08.

Since my commit kept the `rc_sender_id == rc_member_ids[0]` and I did not get an IndexError during import, I guess at least one recipient always exists so there's nothing else to do here now.